.github/workflows/ocicrypt_rs_build.yml

```
@@ -45,104 +39,71 @@ jobs:
         with:
           fetch-depth: 1
 
-      - name: Install Rust toolchain (${{ matrix.rust }})
-        uses: actions-rs/toolchain@v1
+      - name: Install Rust toolchain
+        uses: actions-rust-lang/setup-rust-toolchain@v1
         with:
-          profile: minimal
-          toolchain: ${{ matrix.rust }}
-          override: true
           components: rustfmt, clippy
 
       - name: Install tonic's protoc dependencies
         run: |
           sudo apt install -y protobuf-compiler libprotobuf-dev
 
       - name: Run cargo build
-        uses: actions-rs/cargo@v1
-        with:
-          command: build
-          args: -p ocicrypt-rs --all-features
+        run:
+          cargo build -p ocicrypt-rs --all-features
```

afaiu this will not work. the statement needs to be in the same line without > or | prefixes

```suggestion
        run: cargo build -p ocicrypt-rs --all-features
```